FirstProject/npm-debug.log

```diff
@@ -0,0 +1,25 @@
+0 info it worked if it ends with ok
```

Comment:
Once a file is committed, it is irrelevant whether or not it is included in `.gitignore` after the fact. What you need to do then is to instruct `git` to remove the file(s) from the repository, without physically removing the files, as described [here](https://stackoverflow.com/questions/13541615/how-to-remove-files-that-are-listed-in-the-gitignore-but-still-on-the-repositor).